### PR TITLE
feat(web/applications): add representations pagination

### PR DESCRIPTION
**Pull request by jonathangoulding**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/search-results/applications-search.controller.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export async function searchApplications(req, response) {
 		});
 	}
 
-	const pageSize = 50;
+	const pageSize = 1;
```
Comment (francescobrenci-AR) [inline apps/web/src/server/applications/search-results/applications-search.controller.js:29]: why this change to the search page?

**Comment by francescobrenci-AR** [comment]
There are some files added to the `web/src/common` folder - are this intended to be used in both Server and client code?

if not they should live in the `web/src/server/applications/common/components`

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

File: apps/web/src/server/applications/search-results/applications-search.controller.js (hunk 1 above)
Comment (jonathangoulding) [reply apps/web/src/server/applications/search-results/applications-search.controller.js:29]: will put back - was testing how the pagination works

**Review by lutamKC — APPROVED** [review]
